editor/src/main/com/mbrlabs/mundus/editor/input/FreeCamController.kt

```diff
@@ -95,7 +97,16 @@ class FreeCamController : InputAdapter() {
 
                 camera!!.direction.rotate(camera!!.up, deltaX)
                 tmp.set(camera!!.direction).crs(camera!!.up).nor()
-                camera!!.direction.rotate(tmp, deltaY)
+
+                // Resolves Gimbal Lock : https://github.com/libgdx/libgdx/issues/4023
+                val oldPitchAxis = tmp.set(camera!!.direction).crs(camera!!.up).nor()
```

Comment:
What do you think about gdx's Pools? I think here you can use it to obtain Vector3 objects and free them. And the application could use these objects somewhere else. A bit less memory usage.

Reply:
Vector3 pool? Hm yes I used them in my last jam game, and it did seem to work well but that was the first time I tried it. I am open to implementing them on Mundus but not for this PR since it would be out of the scope.

Reply:
I think if a class uses it every frame then use it without pool. Like this case.